### ISSUE TITLE
Invalid reach collision boxes replication

### ISSUE THREAD

**Issue by interable** [GrimAnticheat/Grim#1489]
### Describe the bug and how to replicate it

```java
this.oldPacketLocation = newPacketLocation;
this.newPacketLocation = new ReachInterpolationData(player, oldPacketLocation.getPossibleLocationCombined(), trackedServerPosition, this);
```

so because of ```oldPacketLocation.getPossibleLocationCombined()``` replicated hitBox can grow infinitely, often hitBox can over grown to 8-16 X/Z blocks, only resets when tracker teleport packet happen.

you can test that by just by adding that part of code in Reach class:
```java
        // print the box
        System.out.println("X: " + (targetBox.maxX - targetBox.minX) + " Y: " + (targetBox.maxY - targetBox.minY) + " Z: " + (targetBox.maxZ - targetBox.minZ));

        double minDistance = Double.MAX_VALUE;
 ```

### Grim version

[0fb9fffb1e9b813a42bfdea3bf4a08a1a87b05d6](https://github.com/GrimAnticheat/Grim/tree/0fb9fffb1e9b813a42bfdea3bf4a08a1a87b05d6)

### Server version

1.8 paper

### Plugins

none

**Comment by interable** [comment]
that bug happen even before https://github.com/GrimAnticheat/Grim/pull/1487 FYI

**Comment by 0-x-2-2** [comment]
yes, this is related to standing 0.03 as-well, grim v3 has mitigations for this. `getPossibleLocationCombined` is supposed to be between those two positions because of transaction splitting. growing infinitely is not exactly correct but it can grow to very large AABBs since it combines the boxes together.

**Comment by Axionize** [comment]
> ### Describe the bug and how to replicate it
> this.oldPacketLocation = newPacketLocation;
> this.newPacketLocation = new ReachInterpolationData(player, oldPacketLocation.getPossibleLocationCombined(), trackedServerPosition, this);
> so because of `oldPacketLocation.getPossibleLocationCombined()` replicated hitBox can grow infinitely, often hitBox can over grown to 8-16 X/Z blocks, only resets when tracker teleport packet happen.
> 
> you can test that by just by adding that part of code in Reach class:
> 
>         // print the box
>         System.out.println("X: " + (targetBox.maxX - targetBox.minX) + " Y: " + (targetBox.maxY - targetBox.minY) + " Z: " + (targetBox.maxZ - targetBox.minZ));
> 
>         double minDistance = Double.MAX_VALUE;
> ### Grim version
> [0fb9fffb1e9b813a42bfdea3bf4a08a1a87b05d6](https://github.com/GrimAnticheat/Grim/tree/0fb9fffb1e9b813a42bfdea3bf4a08a1a87b05d6)
> 
> ### Server version
> 1.8 paper
> 
> ### Plugins
> none

if its growing to 8-16 blocks its because you're on 1.8 server that doesn't have explicit flush enabled and the 2nd transaction packet. Set `Reach.enable-post-packet: true` in your config and `-Dpaper.explicit-flush=true` to your server's startup flags.

**Comment by Axionize** [comment]
Since there's no response and I've already giving a solution, I'll be closing.
